src/renderer/services/dex.js

```diff
@@ -167,30 +167,7 @@ export default {
               trades: res.data.trades,
               getBars: this.getTradeHistoryBars,
             };
-            const todayCutoff = moment().startOf('day').unix();
-            const todayTrades = _.filter(history.trades, (trade) => {
-              return trade.tradeTime >= todayCutoff;
-            });
-            if (todayTrades.length > 0) {
-              history.close24Hour = todayTrades[0].price;
-              history.open24Hour = todayTrades[todayTrades.length - 1].price;
-              history.low24Hour = _.minBy(todayTrades, 'price').price;
-              history.high24Hour = _.maxBy(todayTrades, 'price').price;
-              history.volume24Hour = _.sumBy(todayTrades, 'quantity');
-              history.change24HourPercent = Math.round(((history.close24Hour - history.open24Hour)
-                / history.open24Hour) * 10000) / 100;
-              history.change24Hour = history.close24Hour - history.open24Hour;
-            } else {
-              if (history.trades.length > 0) {
-                history.close24Hour = history.trades[0].price;
-                history.open24Hour = history.trades[0].price;
-                history.low24Hour = history.trades[0].price;
-                history.high24Hour = history.trades[0].price;
-              }
-              history.volume24Hour = 0;
-              history.change24Hour = 0;
-              history.change24HourPercent = 0;
-            }
```

Comment:
This logic should stay, that way the trade history has all of these values and we don't have to calculate these values every time we need to use them.